controller/appcontroller.go

```diff
@@ -1480,6 +1478,9 @@ func (ctrl *ApplicationController) processAppRefreshQueueItem() (processNext boo
 			log.Errorf("Recovered from panic: %+v\n%s", r, debug.Stack())
 		}
 		ctrl.appRefreshQueue.Done(appKey)
+		// In most cases we want to have app operation update happen after the sync, so
```

Comment:
_Commenting on a line so we can have a discussion in a thread._
Can you share metrics